modules/avatar/react/lib/Avatar.tsx

```diff
@@ -68,26 +67,28 @@ const StyledContainer = styled('button', {
     boxSizing: 'border-box',
     overflow: 'hidden',
     borderRadius: borderRadius.circle,
-  },
-  ({size}) => ({
-    height: size,
-    width: size,
-  }),
-  ({variant, onClick}) => ({
-    cursor: onClick ? 'pointer' : 'default',
     '&:not([disabled])': {
       '&:focus': {
         outline: 'none',
-        ...(variant === AvatarVariant.Dark ? focusRing({separation: 2}) : focusRing()),
+        ...focusRing({separation: 2}),
```

Comment:
Note for posterity: I realized that there are no guarantees of the avatar color regardless of the variant so it doesn't make sense for this to be conditional